### PR TITLE
Filters for swear words in input or output

### PR DESCRIPTION
**Pull request by RehanSD**
Makes sure that inputs with curse words can't be provided. Censors swear words in responses. Also fixes `clipper.py` so that it can be run as a faux backend for any of our currently deployed models.

### CODE REVIEW

File: js/src/Components/NLPGenerator.tsx [hunk 1]
```diff
@@ -115,6 +115,9 @@ interface SingleTextProp {
   modelName: string;
 }
 
+var Filter = require('bad-words'),
```
Comment (simon-mo) [inline js/src/Components/NLPGenerator.tsx]:
```suggestion
import Filter from 'bad-words'
```

Use import semantics for consistency

Comment (simon-mo) [inline js/src/Components/NLPGenerator.tsx]: Also, it should be `const Filter`

File: js/src/Components/NLPInferencePage.tsx [hunk 2]
```diff
@@ -13,16 +13,21 @@ interface InferecePageProp {
 
 const defaultPhrase = "Serverless ";
 
+var Filter = require('bad-words'),
```
Comment (simon-mo) [inline js/src/Components/NLPInferencePage.tsx]: same comment in js/src/Components/NLPGenerator.tsx

File: js/src/Components/NLPInferencePage.tsx [hunk 3]
```diff
@@ -13,16 +13,21 @@ interface InferecePageProp {
...
   const [textCache, setTextCache] = useState<string>(defaultPhrase);
   const [textIDCounter, setTextIDCounter] = useState(0);
+  const [alertComponent, setAlertComponent] = useState<JSX.Element>();
 
   const removeTextComp = (val: number) => {
     setAddedTexts(addedTexts => addedTexts.filter(v => v.id !== val));
   };
 
   function createTextRow(result: string) {
+    var profane = filter.isProfane(result);
```
Comment (simon-mo) [inline js/src/Components/NLPInferencePage.tsx]: `const` or `let`

File: js/src/Components/NLPGenerator.tsx (hunk 1 above)
Comment (RehanSD) [reply js/src/Components/NLPGenerator.tsx]: Done.

Comment (RehanSD) [reply js/src/Components/NLPGenerator.tsx]: Taken care of by above.

File: js/src/Components/NLPInferencePage.tsx (hunk 2 above)
Comment (RehanSD) [reply js/src/Components/NLPInferencePage.tsx]: Taken care of by above.

File: js/src/Components/NLPInferencePage.tsx (hunk 3 above)
Comment (RehanSD) [reply js/src/Components/NLPInferencePage.tsx]: Done!

**Pull request merged**